

export const CreateUser = async () => {
    try {

    } catch (error) {
        throw new Error()
    }
}
export const UpdateUser = async () => {
    try {

    } catch (error) {
        throw new Error()
    }
}
export const DeleteUser = async () => {
    try {

    } catch (error) {
        throw new Error()
    }
}
export const getUser = async () => {
    try {

    } catch (error) {
        throw new Error()
    }
}

export const GetAllusers = async () => {
    try {

    } catch (err) {
        throw new Error()
    }
}
